pkg/agent/flowexporter/connections/conntrack_linux_test.go

```diff
@@ -201,3 +218,69 @@ func TestConnTrackOvsAppCtl_GetMaxConnections(t *testing.T) {
 	assert.NoErrorf(t, err, "GetMaxConnections function returned error: %v", err)
 	assert.Equal(t, expMaxConns, maxConns, "The return value of GetMaxConnections function should be equal to the previous hard-coded value")
 }
+
+func TestNetLinkFlowToAntreaConnection(t *testing.T) {
+	// Create new conntrack flow with status as assured connection.
+	netlinkFlow := &conntrack.Flow{
+		TupleOrig: conntrackFlowTuple, TupleReply: conntrackFlowTuple, TupleMaster: conntrackFlowTuple,
+		Timeout: 123, Status: conntrack.Status{Value: 0x4}, Mark: 0x1234, Zone: 2,
+		Timestamp: conntrack.Timestamp{Start: time.Date(2020, 7, 25, 8, 40, 8, 959000000, time.UTC)},
+	}
+	tuple, _ := makeTuple(&conntrackFlowTuple.IP.SourceAddress, &conntrackFlowTuple.IP.DestinationAddress, conntrackFlowTuple.Proto.Protocol, conntrackFlowTuple.Proto.SourcePort, conntrackFlowTuple.Proto.DestinationPort)
+	expectedAntreaFlow := &flowexporter.Connection{
+		Timeout:                 netlinkFlow.Timeout,
+		StartTime:               netlinkFlow.Timestamp.Start,
+		IsPresent:               true,
+		Zone:                    2,
+		StatusFlag:              0x4,
+		Mark:                    0x1234,
+		TupleOrig:               tuple,
+		TupleReply:              tuple,
+		OriginalPackets:         netlinkFlow.CountersOrig.Packets,
+		OriginalBytes:           netlinkFlow.CountersOrig.Bytes,
+		ReversePackets:          netlinkFlow.CountersReply.Packets,
+		ReverseBytes:            netlinkFlow.CountersReply.Bytes,
+		SourcePodNamespace:      "",
+		SourcePodName:           "",
+		DestinationPodNamespace: "",
+		DestinationPodName:      "",
+	}
+
+	antreaFlow := NetlinkFlowToAntreaConnection(netlinkFlow)
+	// Just add the stop time directly as it will the current time during the execution
```

Comment:
sentence doesn't make sense. Did you mean "as it will be set to the time of day at which the function was executed"?

Reply:
done